### ISSUE TITLE
Correct definition of the loss function for neural network potential

### ISSUE THREAD

**Issue by hghcomphys** [hghcomphys/jaxip#15]
The loss function for the NNP is given by combining of the energy and force components respect to the corresponding (DFT) reference data. An equation for the loss function can be found in `Eq. 47` of [this](https://pubs.acs.org/doi/10.1021/acs.chemrev.0c00868) paper.

There, the total energy __per atom__ is used in order to make loss function scalable respect to the size of system (number of atoms).
However, the force components is also suggested to be normalized to the number of atoms which does not make sense as the each force component is already a per atom quantity!
